namespace.py

```diff
@@ -575,7 +585,7 @@ def check(self, val):
 
 
 
-class DictOrSafeDict(ObjectProcessor):
+class DictOrSafeDict(ValueProcessor):
   """Allows SafeDict objects or regular dict objects."""
 
```

Comment:
Confirm failing unit tests when sub-classing `ObjectProcessor` and passing unit tests when sub-classing `ValueProcessor` as claimed in commit message of 7222b4bc52d3a5966e67e7e353b70c28dee4c5f3.
```
RUNNABLE lukp$ python utf.py -f ut_repyv2api_virtualnamespace-eval.py ut_repyv2api_virtualnamespacecontextsafety.py
Testing module: repyv2api
	Running: ut_repyv2api_virtualnamespace-eval.py              [ PASS ]
	Running: ut_repyv2api_virtualnamespacecontextsafety.py      [ PASS ]
```